go.mod

```diff
@@ -1,17 +1,19 @@
 module knative.dev/func
 
-go 1.22
+go 1.22.0
+
+replace k8s.io/client-go => k8s.io/client-go v0.30.1
```

Comment:
@dsimansk for some reason I had to pin, k8s.io/client-go otherwise I got errors like:
```
go: knative.dev/func/pkg/k8s imports
	k8s.io/client-go/dynamic: package k8s.io/client-go/dynamic provided by k8s.io/client-go at latest version v0.30.1 but not at required version v1.5.2
go: knative.dev/func/pkg/k8s imports
	k8s.io/client-go/kubernetes: package k8s.io/client-go/kubernetes provided by k8s.io/client-go at latest version v0.30.1 but not at required version v1.5.2
go: knative.dev/func/pkg/k8s imports
	k8s.io/client-go/kubernetes/scheme: package k8s.io/client-go/kubernetes/scheme provided by k8s.io/client-go at latest version v0.30.1 but not at required version v1.5.2
go: knative.dev/func/pkg/k8s imports
```

Reply:
Tekton CLI seems to be fine: https://github.com/tektoncd/cli/blob/release-v0.37.0/go.mod#L41.

Is it some other package then?

Reply:
Its github.com/openshift-pipelines/pipelines-as-code, I added also comment about it above the replace.